src/ledger/insert_shred.zig

```diff
@@ -617,6 +625,47 @@ pub const ShredInserter = struct {
         };
     }
 
+    /// QUESTION: Okay to reuse this documentation from Agave?
```

Comment:
I assume this documentation was used during the migration of logic, and it was forgotten to actualize